README.md

```diff
@@ -24,6 +25,8 @@ jobs:
         with:
           app_id: ${{ vars.APP_ID }}
           private_key: ${{ secrets.PRIVATE_KEY }}
+          owener: ${{ github.repository_owner }}
+          repositories: repo1,repo2
```

Comment:
remove from this usage example, see above

Reply:
Done.